kernel/src/main/java/org/kframework/compile/ResolveFunctionWithConfig.java

```diff
@@ -233,9 +233,6 @@ public Sentence resolveConfigVar(Sentence s) {
     }
 
     public Sentence resolve(Module m, Sentence s) {
-        if (ExpandMacros.isMacro(s)) {
-            return s;
-        }
```

Comment:
So now we are expanding macros on macro rules? Just want to make sure that I understand what this is doing.

Reply:
This is applying the function context pass to macro rules.